### PR TITLE
Function Add: add graph classification model SAGPool

### PR DESCRIPTION
**Pull request by DesmonDay**

### CODE REVIEW

File: examples/SAGPool/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+python main.py --dataset_name NCI1 --learning_rate 0.001 --weight_decay 0.00001
+
+python main.py --dataset_name NCI109 --learning_rate 0.0005 --hidden_size 64 --weight_decay 0.0001 --patience 200 
+```
+
+## Hyperparameters
+
+- seed: random seed
+- batch\_size: the number of batch size
+- learning\_rate: the number of learning rate
```
Comment (wawltor) [inline examples/SAGPool/README.md:30]: the number of learning rate 感觉表述怪怪的，直接写learning rate of optimizer
